### DISCUSSION TITLE
Using a CSV to set payload rather than HTTP or MQTT

### DISCUSSION THREAD

**Discussion by Mugheeera** [eclipse-ditto/ditto#1606] [Q&A]
Hey eveyone,

I need to make a digital twin for a college project. I have the historical data for the CPS in question but do not have access to the real time data. Is it possible to set the payload using a CSV or a python script? Is Ditto the right framework for such a task? Thank you.

**Comment by thjaeckle** [comment]
Hi.
Not without custom code which you would have to write in order to get the data to Ditto Protocol Format.

There is no built in csv parsing in Ditto.
